egs/fisher_english/s5/local/fisher_train_lms_pocolm.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# Copyright 2016  Vincent Nguyen
+#           2016  Johns Hopkins University (author: Daniel Povey)
+#           2017  Vimal Manohar
+# Apache 2.0
+#
+# It is based on the example scripts distributed with PocoLM
```

Comment:
It would be nice if the comment were to explain how this differs from fisher_train_lms.sh.  I don't understand why this script was added.